packages/additionalapi/src/database/AdditionalQueries.ts

```diff
@@ -37,8 +31,9 @@ export class AdditionalQueries {
             query = makeQueryNodeTreeForIdList(nodeIdList, depthLimit)
             return { status: 410, query: query, queryResult: await this.context.dbConnection.query(query) }
         } catch (e) {
-            console.log("Exception catched in getNodeTree(): " + e.message)
-            return { status: 200, query: query, queryResult: e.message }
+            const error = asError(e)
+            console.log("Exception catched in getNodeTree(): " + error.message)
+            return { status: 200, query: query, queryResult: error.message }
```

Comment:
Could we use a constant instead of the numeric value 200?

Reply:
I have added enumerations for the HTTP codes and use them as stats.